### ISSUE TITLE
[Request] Change color for keywords and types

### ISSUE THREAD

**Issue by jahirfiquitiva** [fabian-hiller/vscode-pace-theme#2]
Hey, I wanted to ask if it's possible to change the colors used for `const` and types, as it's close to the green for class names which makes it a bit confusing.

Maybe the `Pink` or `Fuchsia` colors could fit the palette plus make those texts more distinguishable.

Thanks!

<img width="343" alt="Shot 2022-09-04 at 13 54 22@2x" src="https://user-images.githubusercontent.com/10360816/188329117-ce14421a-1b41-4aec-90eb-667c8b5307cf.png">

Even the preview image has this "issue"
![Dark theme preview image](https://github.com/fabian-hiller/vscode-pace-theme/raw/main/dark-mode.jpg)

**Comment by fabian-hiller** [comment]
Thank you for your feedback! I am aware of the slight color difference. Personally, it hasn't bothered me so far. I will take a look at it with `Fuchsia` in the coming days.

**Comment by jahirfiquitiva** [comment]
@fabian-hiller yeah, it isn't that bad, I just think it would be good to make the difference more noticeable 😅

**Comment by fabian-hiller** [comment]
I tested it with the color `Fuchsia `. For my taste, it ruins the overall look. Therefore, I would like to stay with the previous color for now.

**Comment by jahirfiquitiva** [comment]
@fabian-hiller Any chance of having it as a variant? 🙏 😅

**Comment by fabian-hiller** [comment]
Yes, can you think of a suitable name?

**Comment by jahirfiquitiva** [comment]
@fabian-hiller what do you think about `Pop`? 🤔

**Comment by fabian-hiller** [comment]
This could work. In the long term, I prefer to have one or at most two variants of Dark and Light again, which correspond to my ideas. So it might be a good idea for you to fork the project and create your own theme out of it. On YouTube you can find videos about it. Have a great start to the week!

**Comment by BaiJiang007** [comment]
It's difficult to distinguish keywords and texts😭️

**Comment by fabian-hiller** [comment]
Can you send me a screenshot and highlight the text you would change the color of? Do you have a color recommendation?

**Comment by jahirfiquitiva** [comment]
@fabian-hiller 

I think we mean these texts:
![Image 188329117](https://github.com/fabian-hiller/vscode-pace-theme/assets/10360816/7be631cb-11d1-4643-bd47-fa20bcf8bf6a)

The colors shades are really similar.

Here are some color suggestions that can be used for one of those (maybe the `const` keywords) for the different themes

| Light | Light+ | Dark |
| ------- | ------- | ------- |
| `#5f27cd` | `#341f97` | `#ff9ff3` |
| `#786fa6` | `#574b90` | `#f8a5c2` |
| `#6c5ce7` | `#4C40A2` | `#B5AFFE` |
| `#5851DA` | `#4834d4` | `#ED9AFE` |

**Comment by BaiJiang007** [comment]
![image](https://github.com/fabian-hiller/vscode-pace-theme/assets/129240266/c683045f-cb9f-4726-9e51-18f1dd718625)
When I use these variables, I cannot determine by color whether they have been declared, as they are all white like texts. This creates a significant usage burden

**Comment by fabian-hiller** [comment]
@jahirfiquitiva I am still not quite sure what you mean. Do you think the color of `const` and `fetchSelfFeed` are too similar?

@BaiJiang007 what color should it be? Blue? Can you create a PR?

**Comment by jahirfiquitiva** [comment]
@fabian-hiller **yes**, the colors of `const` and `fetchSelfFeed` are too similar

**Comment by fabian-hiller** [comment]
Thanks for your feedback. Personally it doesn't bother me and at the moment I don't have time to work on it as I'm focused on [Valibot](https://valibot.dev/). But feel free to fork and customize this theme.
